routes/resources/favourites.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * All routes for favourites are defined here
+ * Since this file is loaded in server.js into api/widgets,
+ *   these routes are mounted onto /favourites
+ * See: https://expressjs.com/en/guide/using-middleware.html#middleware.router
+ */
+
+const express = require("express");
+const router = express.Router();
+
+
+const queryGetFavsByUserID = function (db, userID) {
+  let query = `SELECT * FROM favourites WHERE user_id = $1`
+  return db.query(query, [userID]).then((response) => {
+    return response.rows
+  })
+}
+
+module.exports = (db) => {
+  router.get("/", (req, res) => {
+    queryGetFavsByUserID(db, req.query.userID)
```

Comment:
req.query uses a query string (such as `?userID=1`) at the end of the URL, but we may want to use someting like req.params instead to access favourites directly. this would change our route to `/:userID`

Reply:
(params vs query)